src/base/utils/fs.h

```diff
@@ -67,6 +67,10 @@ namespace Utils
         /* End of Qt4 code */
 
         QString cacheLocation();
+
+        /// Returns either default name for configuration file (QCoreApplication::applicationName())
+        /// or the value, supplied via QBT_CONF_NAME environment variable
+        QString configurationName();
```

Comment:
This function, like some other (cacheLocation, QDesktopServices\*) , needs to be moved to another place, because they sense are not file system utilities. These are shell or environment utilities.